site/class-1-load/sebasschmidt/index.JSON

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
Rename this file to `index.json` (lowercase JSON)!

(let me know if you need help)